docs/micro-service-architecture.md

```diff
@@ -0,0 +1,248 @@
+# Architecture document
+The goal of this document is to provide a clear overview of our microservice infrastructure, explain our choices and provide a clear OpenAPI definition.
+
+## Technologies
+
+#### `Kubernetes`
+
+Our production architecture leverages Kubernetes for container orchestration with Docker as the containerization platform. 
+This powerful combination provides scalability, flexibility, and efficient management of containerized applications.
+
+#### `Docker`
+
+We mianly use docker for development with many docker-composes to repliate the micro services and their databases but as said before we 
+also use Docker as the containerization platform in our kubernetes environment
```

Comment:
Same suggestion as before for the "but as said before we also use Docker as the containerization platform in our Kubernetes environment" part.

```suggestion
We mainly use Docker in the local development environment with Docker Compose to bring up the microservices and their databases, but as said before we also use Docker as the containerization platform in our Kubernetes environment.
```